### PR TITLE
Retry on 500 (e.g. due to deployment)

### PR DESCRIPTION
**Pull request by tanin47**
This fixes: #296

### CODE REVIEW

File: tool/clusterfuzz/commands/reproduce.py [hunk 1]
```diff
@@ -85,15 +86,20 @@ def send_request(url, data):
   return a valid, authorized response or throw an exception."""
   header = common.get_stored_auth_header() or get_verification_header()
   response = None
-  for _ in range(2):
+  for _ in range(5):
```
Comment (inferno-chromium) [inline tool/clusterfuzz/commands/reproduce.py]: Move 5 to global var.

File: tool/clusterfuzz/commands/reproduce.py [hunk 2]
```diff
@@ -85,15 +86,20 @@ def send_request(url, data):
...
             'User-Agent': 'clusterfuzz-tools'},
         allow_redirects=True, data=data)
 
-    if response.status_code == 401:  # The access token expired.
+    # The access token expired.
+    if response.status_code == 401:
       header = get_verification_header()
+    # Internal server error (e.g. due to deployment)
+    elif response.status_code == 500:
+      time.sleep(3)
```
Comment (inferno-chromium) [inline tool/clusterfuzz/commands/reproduce.py]: Move 3 to global var.

File: tool/clusterfuzz/commands/reproduce.py (hunk 1 above)
Comment (tanin47) [inline tool/clusterfuzz/commands/reproduce.py]: Done

File: tool/clusterfuzz/commands/reproduce.py (hunk 2 above)
Comment (tanin47) [inline tool/clusterfuzz/commands/reproduce.py]: Done

**Pull request merged**